Frontend/ui-driver/src/Components/BottomNavBar/Controller.purs

```diff
@@ -24,16 +24,15 @@ data Action = OnNavigate String
 navData :: Int -> BottomNavBarState
 navData activeIndex = {
    activeIndex: activeIndex ,
-   screenName : "driver homeScreen screen",
    navButton: [
     {
       activeIcon: "ny_ic_home_active,https://assets.juspay.in/nammayatri/images/driver/ny_ic_home_active.png",
       defaultIcon: "ny_ic_home_inactive,https://assets.juspay.in/nammayatri/images/driver/ny_ic_home_inactive.png",
       text: "Home"
     },
     {
-      activeIcon: "ny_ic_cab_active,https://assets.juspay.in/nammayatri/images/driver/ny_ic_cab_active.png",
-      defaultIcon: "ny_ic_cab_inactive,https://assets.juspay.in/nammayatri/images/driver/ny_ic_cab_inactive.png",
+      activeIcon: "ny_ic_rides_active,https://assets.juspay.in/nammayatri/images/driver/ny_ic_rides_active.png",
```

Comment:
Can you update all the new assets to AssetStore and mention the PR in this PR's description??